src/shared/AppTextArea/AppTextArea.jsx

```diff
@@ -18,6 +18,7 @@ export const AppTextArea = ({
   value,
   onChange,
   placeholder = '',
+  name = '',
```

Comment:
better to provide it through `defaultProps`

Reply:
Hi. I changed it.